docs/product/sentry-basics/concepts/tracing/trace-view.mdx

```diff
@@ -4,47 +4,124 @@ sidebar_order: 30
-</Alert>
-
-## Troubleshooting
-
-### Orphan Traces and Broken Subtraces
-
-In a fully instrumented system, a span in one service will connect to a transaction in a subsequent service. For a variety of reasons, a transaction in a service may not arrive in Sentry. When Sentry encounters these types of transactions within a trace, the transactions are linked with a dashed line since they can no longer be directly connected to the root, creating an orphan trace.
-
-In addition, broken subtraces can occur when Sentry receives an error for a trace, but the corresponding transaction does not exist. Such errors are linked using dashed lines and clicking on the row takes you to the corresponding **Issue Details** page.
-
-Also, in these cases you can click "Open In Discover" to see all the events in the trace.
-
-Broken subtraces may be caused by:
-
-- SDK sampling. Setting a sample rate that's too low may prevent the SDK from sending a transaction. We recommend [sending us all of your transaction data](/product/performance/retention-priorities/#deciding-on-your-sdk-sample-rate).
-- [Ad blockers](/platforms/javascript/troubleshooting/#dealing-with-ad-blockers) may prevent transactions in browsers being sent, but HTTP requests to backend projects will still create child transactions
-- [Rate-limiting](/product/accounts/quotas/#limiting-events) on a project may cause only some events to be sent to Sentry
-- [Project permissions](/product/accounts/membership/#restricting-access) may mean you do not have access to transactions in another project
-- [Differences in quota limits](/product/accounts/quotas/) between transactions and errors. When a quota limit is reached - for example, for transactions - an error is received, but corresponding transaction is not.
-- Exceeding the span limit. Transactions are associated via the child spans of the parent transaction, but if the number of spans exceed the limit, the association cannot be made
-
-### Multiple Roots
-
-Each trace ID should have only one root, a transaction without any parents. Automatic instrumentation should prevent multiple roots; however, if the trace ID of your transactions is being set using custom instrumentation, you may encounter multiple roots.
+The trace view is designed to be your one-stop-shop when debugging performance or errors. It gives you full context on what was happening when an error or performance issue occurred, all in one place. The waterfall trace view allows you to see everything that may have occurred during a trace, including errors, performance issues, and any profiles that may have been collected.
+
+In addition, the trace view allows you to drill down into the details of a single trace so you can visualize the high level transactions and spans that took place within that trace. This makes debugging slow services, identifying related errors, and rooting out bottlenecks easer and faster without having to navigate around multiple pages of the Sentry product.
```

Comment:
```suggestion
In addition, the trace view allows you to drill down into the details of a single trace so you can visualize the high-level transactions and spans that took place within that trace. This makes debugging slow services, identifying related errors, and rooting out bottlenecks easier and faster, since you no longer have to navigate around multiple pages of the Sentry product.
```